modules/unsupported/wfs-ng/src/main/java/org/geotools/data/wfs/impl/WFSDataAccessFactory.java

```diff
@@ -589,6 +590,22 @@ public T lookUp(final Map params) throws IOException {
                         new WFSFactoryParam<>(name, Integer.class, title, description, 6);
     }
 
+    /**
+     * Optional {@code String} Flag to enabled/disable usage of secured http client wrapper that
+     * will only make requests to allowed urls and paths
+     */
+    public static final WFSFactoryParam<Boolean> SECURED_HTTP_CLIENT;
```

Comment:
Makes some sense to have a parameter... but nothing else has it, e.g., WMS and WMTS cascading do not, coverage readers going through ImageIOExt neither, they all have it always on. At this point I'd suggest to do the same here and see if we need to make it parametric (in a consistent way) later.

Reply:
The WMS and WMTS implementation is in Geoserver, where the parameter and wrapping of HttpClient is done inside HttpStoreInfo abstrat class. Well for ImageIOExt, looks I will have to add new functions that wrap previous one for backward compatability, but then a question arises, what is the backward compatability behavior? Should the evaluation be turned off by default inside ImageIOExt unless requested?

Reply:
will remove this FLAG, same will be done on GeoServer side then.